distribution/docs/src/main/resources/content/_managing/_installing/high-availability-initial-setup-exceptions.adoc

```diff
@@ -5,11 +5,11 @@
 :project: ${branding}
 :order: 00
 
-==== {title}
+== {title}
 
 These steps are handled differently for the initial setup of a Highly Available Cluster.
 
-===== Failover Proxy Integration
+=== Failover Proxy Integration
```

Comment:
❓ Are these header levels meant to be changing in this PR?

Reply:
fixed